src/miniscript/iter.rs

```diff
@@ -125,8 +126,16 @@ impl<Pk: MiniscriptKey, Ctx: ScriptContext> Miniscript<Pk, Ctx> {
     /// `miniscript.iter_pubkeys().collect()`.
     pub fn get_leapk(&self) -> Vec<Pk> {
         match self.node {
-            Terminal::PkK(ref key) | Terminal::PkH(ref key) => vec![key.clone()],
-            Terminal::Multi(_, ref keys) | Terminal::MultiA(_, ref keys) => keys.clone(),
+            Terminal::PkK(ref key) => key.iter().map(|pk| pk.clone()).collect(),
```

Comment:
I have updated the iterator file at: https://github.com/sanket1729/rust-miniscript/tree/pr460

Reply:
You can cherry pick the commits and add them here